code/src/uarm_swift_pro.cpp

```diff
@@ -0,0 +1,46 @@
+#include <uarm_swift_pro.hpp>
+
+namespace r2d2::robot_arm {
+    uarm_swift_pro_c::uarm_swift_pro_c(const r2d2::hardware_usart_c &usart_bus)
```

Comment:
See comments in the header

Reply:
We are at this moment not able to test this change request. We will inform the next team of this needed change.